### ISSUE TITLE
Local test failed because of file-name-linter

### ISSUE THREAD

**Issue by fabian4** [wechaty/docusaurus#1411]
# Here is the result on my local machine when I run `npm run test`
~~~shell
npm run test

> wechaty.js.org@0.10.52 test
> npm-run-all lint test:unit


> wechaty.js.org@0.10.52 lint
> npm-run-all lint:es lint:ts lint:md lint:fn lint:yml


> wechaty.js.org@0.10.52 lint:es
> eslint "src/**/*.ts" "tests/**/*.spec.ts" --ignore-pattern tests/fixtures/

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.5.0

YOUR TYPESCRIPT VERSION: 4.5.2

Please only submit bug reports when using the officially supported version.

=============

> wechaty.js.org@0.10.52 lint:ts
> tsc --isolatedModules --noEmit


> wechaty.js.org@0.10.52 lint:md
> markdownlint --ignore _site/ --ignore node_modules/ --ignore docusaurus/node_modules/ "**/*.md"


> wechaty.js.org@0.10.52 lint:fn
> file-name-linter .file-name-linter.json

 ✖  jekyll\_posts\2016-12-03-welcome-to-wechaty.md does not match /\/[0-9\-a-z]+\./!
 ✖  jekyll\_posts\2016-12-05-ghostcloud-wechaty-docker.md does not match /\/[0-9\-a-z]+\./!
 ✖  jekyll\_posts\2016-12-04-gitchat-lijiarui-interview.md does not match /\/[0-9\-a-z]+\./!
 ✖  jekyll\_posts\2016-12-07-beijing-nodejs-meetup-party.md does not match /\/[0-9\-a-z]+\./!
 ✖  jekyll\_posts\2016-12-08-jaslin-user-case.md does not match /\/[0-9\-a-z]+\./!
 ✖  jekyll\_posts\2016-12-10-try-to-write-wexinrobot.md does not match /\/[0-9\-a-z]+\./!
 ✖  jekyll\_posts\2017-01-01-getting-started-wechaty.md does not match /\/[0-9\-a-z]+\./!
...........

 🏁  Finished linting jekyll/assets
ERROR: "lint:fn" exited with 1.
ERROR: "lint" exited with 1.
~~~

# **Is my typescript version the reason why it always fails ?**

**Comment by huan** [comment]
TL;DR: use Linux to run this script. It does not support Windows yet.

Merge to https://github.com/wechaty/wechaty.js.org/issues/1363#issuecomment-945171826
